autorag/nodes/passagereranker/monot5.py

```diff
@@ -0,0 +1,95 @@
+from typing import List, Tuple
+from uuid import UUID
+
+import torch
+import asyncio
+
+from transformers import T5Tokenizer, T5ForConditionalGeneration
+
+
+prediction_tokens = {
+    'castorini/monot5-base-msmarco': ['▁false', '▁true'],
+    'castorini/monot5-base-msmarco-10k': ['▁false', '▁true'],
+    'castorini/monot5-large-msmarco': ['▁false', '▁true'],
+    'castorini/monot5-large-msmarco-10k': ['▁false', '▁true'],
+    'castorini/monot5-base-med-msmarco': ['▁false', '▁true'],
+    'castorini/monot5-3b-med-msmarco': ['▁false', '▁true'],
+    'castorini/monot5-3b-msmarco-10k': ['▁false', '▁true'],
+    'unicamp-dl/mt5-base-en-msmarco': ['▁no', '▁yes'],
+    'unicamp-dl/ptt5-base-pt-msmarco-10k-v2': ['▁não', '▁sim'],
+    'unicamp-dl/ptt5-base-pt-msmarco-100k-v2': ['▁não', '▁sim'],
+    'unicamp-dl/ptt5-base-en-pt-msmarco-100k-v2': ['▁não', '▁sim'],
+    'unicamp-dl/mt5-base-en-pt-msmarco-v2': ['▁no', '▁yes'],
+    'unicamp-dl/mt5-base-mmarco-v2': ['▁no', '▁yes'],
+    'unicamp-dl/mt5-base-en-pt-msmarco-v1': ['▁no', '▁yes'],
+    'unicamp-dl/mt5-base-mmarco-v1': ['▁no', '▁yes'],
+    'unicamp-dl/ptt5-base-pt-msmarco-10k-v1': ['▁não', '▁sim'],
+    'unicamp-dl/ptt5-base-pt-msmarco-100k-v1': ['▁não', '▁sim'],
+    'unicamp-dl/ptt5-base-en-pt-msmarco-10k-v1': ['▁não', '▁sim'],
+    'unicamp-dl/mt5-3B-mmarco-en-pt': ['▁', '▁true'],
+    'unicamp-dl/mt5-13b-mmarco-100k': ['▁', '▁true'],
+}
+
+
+def monot5(queries: List[str], contents_list: List[List[str]],
+           scores_list: List[List[float]], ids_list: List[List[UUID]],
+           model_name: str = 'castorini/monot5-3b-msmarco-10k') -> List[Tuple[List[str]]]:
```

Comment:
Why return `List[Tuple[List[str]]]`
According to gunn structure, its return must be 
```
- contents: List[str]
-  ids: List[str]
- scores: List[float]
```